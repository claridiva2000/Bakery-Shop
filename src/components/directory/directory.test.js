import React from 'react';
import renderer from 'react-test-renderer';
import Directory from './directory.component';

it('should render Directory component', () => {
  const component = renderer.create(<Directory />).toJSON();
  expect(component).toMatchSnapshot();
});
