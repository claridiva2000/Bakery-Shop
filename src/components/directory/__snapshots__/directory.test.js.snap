// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Directory component 1`] = `
<div
  className="directory"
>
  <h1>
    Products
  </h1>
  <div
    className="directory-items"
  >
    <div
      className="directory-item"
    >
      <div
        className="image"
      >
        <span>
          <svg>
            cookies.svg
          </svg>
        </span>
      </div>
      <h3>
        Cookies
      </h3>
    </div>
    <div
      className="directory-item"
    >
      <div
        className="image"
      >
        <span>
          <svg>
            cupcakes.svg
          </svg>
        </span>
      </div>
      <h3>
        Cup Cakes
      </h3>
    </div>
    <div
      className="directory-item"
    >
      <div
        className="image"
      >
        <span>
          <svg>
            pudding.svg
          </svg>
        </span>
      </div>
      <h3>
        Puddings
      </h3>
    </div>
    <div
      className="directory-item"
    >
      <div
        className="image"
      >
        <span>
          <svg>
            pies.svg
          </svg>
        </span>
      </div>
      <h3>
        Pies
      </h3>
    </div>
    <div
      className="directory-item"
    >
      <div
        className="image"
      >
        <span>
          <svg>
            cake.svg
          </svg>
        </span>
      </div>
      <h3>
        Cakes
      </h3>
    </div>
  </div>
</div>
`;
